developer_manual/app/container.rst

```diff
@@ -144,9 +144,9 @@ The container works in the following way:
     });
 
 * The **database connection** is returned from the server container
-* Now **AuthorMapper** has all of its dependencies and the object is being returned
+* Now **AuthorMapper** has all of its dependencies and the object is returned
 * **AuthorService** gets the **AuthorMapper** and returns the object
-* **AuthorController** gets the **AuthorService** and finally the controller can be newed and the object is being returned
+* **AuthorController** gets the **AuthorService** and finally the controller can be newed and the object is returned
```

Comment:
You can make a new object with the "new" statement. I guess techos take over language as they like:) The verb "to new" has been created and means "to make a new object using the 'new' statement". And the past tense would be "newed".
To the technical authors: Is that the intended technical meaning here?